teams/wg-safe-transmute.md

```diff
@@ -0,0 +1,18 @@
+name = "safe-transmute"
```

Comment:
Huh, I thought that these had to be named wg-foo ...

Reply:
It's supposed to...

Reply:
@pietroalbini @nikomatsakis whoops that's my fault. Strange that the CI passes....Should I change it?

Reply:
Yes, might as well. It is curious that CI passes though, I got errors when I ran `cargo run -- check` locally